protocols/6aee6e/README.md

```diff
@@ -0,0 +1,31 @@
+# Ammonia ytrium dilution
+
+### Author
+[Opentrons](https://opentrons.com/)
+
+## Categories
+* Nucleic Acid Extraction & Purification
```

Comment:
Can you update this category to `Sample Prep` --> `Plate Filling`?

Afterwards run `make all`

Reply:
Fixed.